config/provider.go

```diff
@@ -111,7 +111,6 @@ var skipList = []string{
 	"aws_alb_target_group$",            // identical with aws_lb_target_group.
 	"aws_alb_target_group_attachment$", // identical with aws_lb_target_group_attachment.
 	"aws_iot_authorizer$",              // failure with unknown reason.
-	"aws_appflow_connector_profile$",   // failure with unknown reason.
```

Comment:
I think we still don't generate this.

Reply:
back "aws_appflow_connector_profile$",   // failure with unknown reason. to config/provider.go